### PR TITLE
update compose banner

### PR DESCRIPTION
**Pull request by aevesdocker**

### CODE REVIEW

File: _includes/compose-eol.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 > **Important**
 >
-> Compose V1 no longer receives updates and will not be available in new releases of Docker Desktop after June 2023.
+> From July 20223 Compose V1 stopped receiving updates. It is also no longer available in new releases of Docker Desktop.
```
Comment (ChrisChinchilla) [inline _includes/compose-eol.md:3]:
```suggestion
> From July 20223 Compose V1 stopped receiving updates. It's also no longer available in new releases of Docker Desktop.
```

File: _includes/compose-eol.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 > **Important**
 >
-> Compose V1 no longer receives updates and will not be available in new releases of Docker Desktop after June 2023.
+> From July 20223 Compose V1 stopped receiving updates. It's also no longer available in new releases of Docker Desktop.
```
Comment (usha-mandya) [inline _includes/compose-eol.md:3]:
```suggestion
> From July 2023 Compose V1 stopped receiving updates. It's also no longer available in new releases of Docker Desktop.
```

Comment (aevesdocker) [reply _includes/compose-eol.md:3]: 🤦‍♀️

**Review by milas — APPROVED** [review]

**Review by glours — APPROVED** [review]
LGTM

**Review by dvdksn — APPROVED** [review]

File: _includes/compose-eol.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 > **Important**
 >
-> Compose V1 no longer receives updates and will not be available in new releases of Docker Desktop after June 2023.
+> From July 2023 Compose V1 stopped receiving updates. It's also no longer available in new releases of Docker Desktop.
```
Comment (laurazard) [inline _includes/compose-eol.md:3]:
```suggestion
> Compose V1 stopped receiving updates **July 2023. It's also no longer available in new releases of Docker Desktop.
```
reads a bit better to me.

**Review by laurazard — APPROVED** [review]
LGTM!

**Pull request merged**